components/mdSerializer.js

```diff
@@ -0,0 +1,9 @@
+// https://stackoverflow.com/a/75635815
+import { serialize } from 'next-mdx-remote/serialize';
+import remarkGfm from 'remark-gfm';
+
+export default function gfmSerialize(source) {
+    return serialize(source, {
+        mdxOptions: { remarkPlugins: [remarkGfm] },
+    })
+}
```

Comment:
@zenVentzi huge thanks for your stackoverflow answer! https://stackoverflow.com/a/75635815/8919142
I spent so long trying to do this, but your stackoverflow answer saved me lol